ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js

```diff
@@ -0,0 +1,261 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+
+(function () {
+
+    function AccordionMixin(Base) {
+        return class extends Base {
+            static NS = "cmp";
+            static IS = "adaptiveFormAccordion";
+            static bemBlock = 'cmp-accordion';
+            static selectors = {
+                self: "[data-" + this.NS + '-is="' + this.IS + '"]'
+            };
+
+
+            static idSuffixes = {
+                item: "-item",
+                button: "-button",
+                panel: "-panel"
+            }
+
+            static cacheKeys = {
+                buttonKey: "button",
+                panelKey: "panel",
+                itemKey: "item"
+            }
+
+            static cssClasses = {
+                button: {
+                    disabled: "cmp-accordion__button--disabled",
+                    expanded: "cmp-accordion__button--expanded"
+                },
+                panel: {
+                    hidden: "cmp-accordion__panel--hidden",
+                    expanded: "cmp-accordion__panel--expanded"
+                }
+            };
+
+            static delay = 100;
+
+            static dataAttributes = {
+                item: {
+                    expanded: "data-cmp-expanded"
+                }
+            };
+
+            constructor(params) {
+                super(params);
+            }
+
+            _getCachedItems() {
```

Comment:
Please use the new way to mention private methods using #

Reply:
This is base class for `Accordion`, if using `#` then child class would not be able to access. So changed to `_`  just to represent it as private.